lib/mm_network.pm

```diff
@@ -103,7 +108,21 @@ sub configure_dhcp {
 }
 
 sub configure_default_gateway {
-    enter_cmd("echo 'default 10.0.2.2 - -' > /etc/sysconfig/network/routes");
+    my (%args) = @_;
+    my $is_nm = $args{is_nm} // is_networkmanager();
+    my $device = $args{device};
+    if ($is_nm) {
+        my $nm_id;
+        my $nm_list = script_output("nmcli -t -f DEVICE,NAME c | grep '$device' | head -n1");
+        ($device, $nm_id) = split(':', $nm_list);
+
+        assert_script_run "nmcli connection modify '$nm_id' ipv4.gateway 10.0.2.2";
+        assert_script_run "nmcli connection down '$nm_id'";
+        assert_script_run "nmcli connection up '$nm_id'";
+    } else {
+        enter_cmd("echo 'default 10.0.2.2 - -' > /etc/sysconfig/network/routes");
```

Comment:
This doesn't change the running network configuration within wicked.
A `wicked ifup all` should do it.
```suggestion
        enter_cmd("echo 'default 10.0.2.2 - -' > /etc/sysconfig/network/routes");
        enter_cmd('wicked ifup all');
```